addons/binding/org.openhab.binding.neato/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="neato"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>Neato Binding</name>
+    <description>This is the binding for Neato.</description>
+    <author>Patrik Wimnell</author>
+    <config-description>
+        <parameter name="email" type="text">
```

Comment:
Not sure about this xml spec, but in HTML5 (Paper UI) you can use `type="email"` for native email validation.  @aounhaider1 does Paper UI handle that scenario?